src/components/gallery-masonry-section.tsx

```
@@ -0,0 +1,28 @@
+import { BLUR_DATA_URL } from '@/constant';
+import type { GalleryMasonrySection } from '@/types';
+import Image from 'next/image';
+
+export default function GalleryMasonrySection({ data }: { data: GalleryMasonrySection }) {
```

Biome (1.9.4)

[error] Shouldn't redeclare 'GalleryMasonrySection'. Consider to delete it or rename it.

'GalleryMasonrySection' is defined here:


(lint/suspicious/noRedeclare)